cypress/integration/invoices/invoices.createCreditInvoice.spec.js

```diff
@@ -0,0 +1,71 @@
+import TopMenu from '../../support/fragments/topMenu';
+import NewInvoice from '../../support/fragments/invoices/newInvoice';
+import NewInvoiceLine from '../../support/fragments/invoices/newInvoiceLine';
+import Invoices from '../../support/fragments/invoices/invoices';
+import TestType from '../../support/dictionary/testTypes';
+import VendorAddress from '../../support/fragments/invoices/vendorAddress';
+import NewFund from '../../support/fragments/finance/funds/newFund';
+import Funds from '../../support/fragments/finance/funds/funds';
+import { getCurrentFiscalYearCode } from '../../support/utils/dateTools';
+import Helper from '../../support/fragments/finance/financeHelper';
+import Transaction from '../../support/fragments/finance/fabrics/newTransaction';
+
+describe('ui-invoices: Credit Invoice creation', () => {
+  const invoice = { ...NewInvoice.defaultUiInvoice };
+  const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
+  const invoiceLine = { ...NewInvoiceLine.defaultUiInvoiceLine };
+  const fund = { ...NewFund.defaultFund };
+  const subtotalValue = 100;
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getOrganizationApi({ query: `name=${invoice.vendorName}` })
+      .then(({ body }) => {
+        invoice.accountingCode = body.organizations[0].erpCode;
+        Object.assign(vendorPrimaryAddress,
+          body.organizations[0].addresses.find(address => address.isPrimary === true));
+      });
+    cy.getBatchGroups()
+      .then(({ body }) => {
+        invoice.batchGroup = body.batchGroups[0].name;
+      });
+    Funds.createFundViaUI(fund)
```

Comment:
do you need to work through GUI instead of API in preconditions?

Reply:
I think it's better from E2E perspective in this particular case, since we're going through the whole flow. Other invoice tests will not contain UI preconditions.